src/current/v23.2/cutover-replication.md

```diff
@@ -0,0 +1,174 @@
+   5 | standbyapplication | replicating | none         | application        | postgresql://user:redacted@host/?options=-ccluster%3Dsystem&sslmode=verify-full&sslrootcert=redacted | 911803003607220225 | 2023-10-26 17:36:52.27978+00 | 2023-10-26 14:36:52.279781+00 |         NULL
+~~~
+
+Run the following from the standby cluster's SQL shell to start the cutover:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER standbyapplication COMPLETE REPLICATION TO LATEST;
+~~~
+
+The `cutover_time` is the timestamp at which the replicated data is consistent. The cluster will revert any data above this timestamp:
+
+~~~
+           cutover_time
+----------------------------------
+  1695922878030920020.0000000000
+(1 row)
+~~~
+
+### Cut over to a point in time
+
+You can control the point in time that the replication stream will cut over to.
+
+To select a [specific time]({% link {{ page.version.version }}/as-of-system-time.md %}) in the past, use:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW VIRTUAL CLUSTER standbyapplication WITH REPLICATION STATUS;
+~~~
+
+The `retained_time` response provides the earliest time to which you can cut over.
+
+{% include_cached copy-clipboard.html %}
+~~~
+id |        name        |     data_state     | service_mode | source_tenant_name |                                                     source_cluster_uri                                               | replication_job_id |        replicated_time        |         retained_time         | cutover_time
+---+--------------------+--------------------+--------------+--------------------+----------------------------------------------------------------------------------------------------------------------+--------------------+-------------------------------+-------------------------------+---------------
+3  | standbyapplication | replicating        | none         | application        | postgresql://{user}:redacted@{hostname}:26257/?options=-ccluster%3Dsystem&sslmode=verify-full&sslrootcert=redacted   | 899090689449132033 | 2023-09-11 22:29:35.085548+00 | 2023-09-11 16:51:43.612846+00 |     NULL
+(1 row)
+~~~
+
+Specify a timestamp:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER standbyapplication COMPLETE REPLICATION TO SYSTEM TIME '-1h';
+~~~
+
+Refer to [Using different timestamp formats]({% link {{ page.version.version }}/as-of-system-time.md %}#using-different-timestamp-formats) for more information.
+
+Similarly, to cut over to a specific time in the future:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER standbyapplication COMPLETE REPLICATION TO SYSTEM TIME '+5h';
+~~~
+
+A future cutover will proceed once the replicated data has reached the specified time.
+
+## Step 2. Completing the cutover
```

Comment:
```suggestion
## Step 2. Complete the cutover
```

Reply:
Chopped that gerund, did the other step too.